### DISCUSSION TITLE
LottieAnimationView.getValue(for keypath: AnimationKeypath, atFrame: AnimationFrameTime?) returning nil

### DISCUSSION THREAD

**Discussion by axel-alphero** [airbnb/lottie-ios#2398] [General]
## Which Version of Lottie are you using?

Lottie 4.4.3

## Expected Behavior 

The `LottieAnimationView.getValue(for keypath: AnimationKeypath, atFrame: AnimationFrameTime?)` returns a value pre-4.x.x (3.5.0)

## Actual Behavior

It now returns `nil`

## Animation JSON

[RoadTest.json](https://github.com/airbnb/lottie-ios/files/15195989/RoadTest.json)

**Comment by calda** [comment]
`getValue(for:atFrame:)` is not supported by the Core Animation rendering engine, but is still supported by the Main Thread rendering engine. You could configure your `LottieAnimationView` to use the Main Thread rendering engine instead.

**Reply by axel-alphero** [reply]
Thank you for that, @calda!
